reana_server/rest/launch.py

```diff
@@ -0,0 +1,108 @@
+# This file is part of REANA.
+# Copyright (C) 2022 CERN.
+#
+# REANA is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+"""REANA-Server launch functionality Flask-Blueprint."""
+
+import json
+import logging
+import traceback
+
+from flask import Blueprint, jsonify
+from marshmallow import Schema
+from webargs import fields
+from webargs.flaskparser import use_kwargs
+
+
+blueprint = Blueprint("launch", __name__)
+
+
+@blueprint.route("/launch", methods=["POST"])
+@use_kwargs(
+    {
+        "url": fields.Url(required=True),
+        "name": fields.Str(),
+        "parameters": fields.Str(),
+    }
+)
+def launch(url, name, parameters="{}"):
+    r"""Endpoint to launch a REANA workflow from URL.
+
+    ---
+    post:
+      summary: Launch workflow from a remote REANA specification file.
+      description: >-
+        This resource expects a remote reference to a REANA specification
+        file needed to launch a workflow via URL.
+      operationId: launch
+      produces:
+        - application/json
+      parameters:
+        - name: url
+          in: query
```

Comment:
The parameters (`url`, `name` and `parameters`) are passed in the body, aren't they? If this is the case, then we also have to add that this endpoint consumes JSON data.

Reply:
Yes, this is exactly what I had in mind!